### PR TITLE
Add macOS shortcuts for undo/redo

### PR DESCRIPTION
**Pull request by msavy**
Want to do a bit more testing on this first before merging, but had a couple of questions (inline)

### CODE REVIEW

File: front-end/studio/src/app/pages/apis/{apiId}/editor/editor.component.ts [hunk 1]
```diff
@@ -193,14 +193,24 @@ export class ApiEditorComponent implements OnChanges, OnInit, OnDestroy, IEditor
...
+        if (ApiEditorComponent.isUndo(event)) {
             this.undoLastCommand();
         }
-        if (event.ctrlKey && event.key === 'y' && !event.metaKey && !event.altKey) {
+        if (ApiEditorComponent.isRedo(event)) {
             this.redoLastCommand();
         }
     }
 
+    private static isUndo(event: KeyboardEvent): boolean {
```
Comment (msavy) [inline front-end/studio/src/app/pages/apis/{apiId}/editor/editor.component.ts]: Would you prefer these in a utility/helper class somewhere? If so, where?

File: front-end/studio/src/app/pages/apis/{apiId}/editor/editor.component.ts [hunk 2]
```diff
@@ -193,14 +193,24 @@ export class ApiEditorComponent implements OnChanges, OnInit, OnDestroy, IEditor
...
+        return (event.ctrlKey && event.key === 'z' && !event.metaKey && !event.altKey) ||
+            (event.metaKey && event.key === 'z' && !event.ctrlKey && !event.altKey);  // macOS: cmd + z
```
Comment (msavy) [inline front-end/studio/src/app/pages/apis/{apiId}/editor/editor.component.ts]: My thoughts on this was that we support both options (macOS shortcuts and windows/linux). So you can do it the Linux/Windows way on Mac if you want.

File: front-end/studio/src/app/pages/apis/{apiId}/editor/editor.component.ts (hunk 1 above)
Comment (msavy) [reply front-end/studio/src/app/pages/apis/{apiId}/editor/editor.component.ts]: Also, leaving them here seems completely reasonable to me, but just wanted to check :)

**Comment by EricWittmann** [comment]
Looks good to me - but I've read that not all browsers necessarily support the "meta" flag on the event.  Not sure if that's still the case - so if you could just test whatever major browsers you have, that'd be great (presumably FF, Chrome, Safari... and obviously IE for Mac!).

PS: kidding about IE for mac

**Comment by msavy** [comment]
Generally seems good— https://developer.mozilla.org/en-US/docs/Web/API/KeyboardEvent/metaKey#Browser_compatibility

Will try it on mobile if my half-dead phone will tolerate it LOL.

**Comment by msavy** [comment]
I've tested this and it seems to work well.

I forgot to sign the commit, will can push a signed version if you wish.

**Comment by EricWittmann** [comment]
It's not required, just recommended as best practice.  No worries.

**Comment by msavy** [comment]
Yes, I normally have it, just forgot to set it up for this repo. 

Although, if you merge via the web UI it generally destroys the signature anyway!

**Pull request merged**